tests/snapshots/tests.contrib.asyncpg.test_asyncpg.test_cursor_manual.json

```diff
@@ -0,0 +1,48 @@
+[[
+  {
+    "name": "postgres.connect",
+    "service": "postgres",
+    "resource": "postgres.connect",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "sql",
+    "meta": {
+      "db.name": "postgres",
+      "db.user": "postgres",
+      "out.host": "127.0.0.1",
+      "runtime-id": "1cac5ff8eab94f03b9fd38a4755df271"
+    },
+    "metrics": {
+      "_dd.agent_psr": 1.0,
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "out.port": 5432,
+      "system.pid": 66764
+    },
+    "duration": 5445000,
+    "start": 1646993027327896000
+  }],
+[
+  {
+    "name": "postgres.cursor",
```

Comment:
🤔 we don't want this as `postgres.query` ?

Reply:
since the name is different from `postges.query` these queries won't get grouped with the others and the metrics won't get aggregated together. should they?

Reply:
yeah I think I wanted to differentiate since the query isn't actually executed when the cursor is created (but other stuff happens)

Reply:
this is only creation of the cursor and doesn't actually execute the query/get results?

Reply:
decided to remove it